packages/expo-image/ios/ImageModule.swift

```diff
@@ -50,7 +50,8 @@ public final class ImageModule: Module {
       }
 
       Prop("blurRadius") { (view, blurRadius: Double?) in
-        view.blurRadius = blurRadius ?? .zero
```

Comment:
<details>
  <summary>screenshots</summary>
![Simulator Screenshot - iPhone 15 Pro Max - 2024-06-11 at 22 35 52](https://github.com/expo/expo/assets/1566403/f0d59141-464a-4599-96c6-b873268aadd3)
![Simulator Screenshot - iPhone 15 Pro Max - 2024-06-11 at 22 35 17](https://github.com/expo/expo/assets/1566403/61e1d5e6-4680-4ce1-86de-7bf6365971e4)

</details>

Reply:
Interesting 🤔 It looks more like the RN Image, so I'm fine with this change. Can you add a comment explaining why we're dividing the radius by 2?

Reply:
I'd love to give a better explanation but the truth is that 2 is a "magic number" in this case. https://developer.apple.com/documentation/coreimage/cigaussianblur is used on iOS to achieve the effect. It'd need some more digging to find the cause, prob not worth it :).